modules/ROOT/pages/penetration-testing-policies.adoc

```diff
@@ -9,40 +9,65 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
-If you are deploying applications to xref:index.adoc[Runtime Manager] or APIs on xref:api-manager::index.adoc[API Manager] and your company's security policies require that you run penetration testing on them, there is a way for you to run these. Given that such tests can interfere with other tenants, penetration testing is only allowed on your own workers. It is not allowed for any other services in the Anypoint Platform.
+If your company security policies require that you perform vulnerability and penetration testing 
+before deploying to Anypoint Platform, you must request a Security Assessment before proceeding 
+with your testing on Anypoint Platform.
 
-External vulnerability scans, via an Approved Scanning Vendor (ASV), as described in section 11.2.2 of the PCI standard may be performed at will and do not require prior approval. Penetration testing as described by section 11.3.1 of the PCI standard does require prior approval from MuleSoft.
+Because these tests can interfere with other tenants, MuleSoft allows penetration testing on only your workers
+but not for other Anypoint Platform services.
 
-CloudHub Workers, by default, do not use static IPs and cannot be properly tested as their IPs may change. Please enable static IPs, as described in xref:deploying-to-cloudhub.adoc#static-ips[Static IPs].
+== Prerequisites
+
+Before penetration testing, enable static IPs for the app.
+
+By default, CloudHub workers do not use static IP addresses, so you can't test them because their IP addresses might change.
+For information about how to enable static IPs, see xref:deploying-to-cloudhub.adoc#static-ips-tab-settings[Static IPs Tab Settings].
+
+== Request a Security Assessment 
+
+To request a Security Assessment, follow the steps in https://help.salesforce.com/articleView?id=000336416&type=1&mode=1[Salesforce Security vulnerability assessment and penetration test]:
+
+. https://help.salesforce.com/articleView?id=000322336&type=1&mode=1[Complete the Security Assessment submission form]. 
+. Assign a security resource at your company to review and validate your findings.
++
+[IMPORTANT]
+You must complete this step before following up with Salesforce. 
++
+Use the following documents to identify common false positives or security issues related to settings:
++
+--
+** https://help.salesforce.com/articleView?id=Salesforce-Platform-Security-FAQs&type=1&language=en_US[Platform Security FAQ]
+** https://help.salesforce.com[Salesforce Help]
+--
+
+. Send your completed Security Assessment form to mailto:securityassessment@salesforce.com[securityassessment@salesforce.com].
+
+. Send any outstanding security vulnerability findings to mailto:security@salesforce.com[security@salesforce.com] and include the following information in your email:
```

Comment:
Outstanding security vulnerability findings are not documented in the SA form?
Why are these two things separate?
When did the audience perform all the steps required to obtain the info in this email message? As part of the SA form, or separately.

I think I am stumbling most on having two separate reporting channels: the SA form and then some very complicated-sounding tests, findings, independent assessments, tests to reproduce, explanations....